Scripts/Python/ki/__init__.py

```diff
@@ -5516,6 +5517,7 @@ def ProcessNotifyMini(self, control, event):
             chatArea.moveCursor(PtGUIMultiLineDirection.kBufferEnd)
             # Hide the chat scroll buttons (should be nothing in chat area yet anyhow).
             chatArea.disableScrollControl()
+            chatArea.setMargins(left=2, right=2)
```

Comment:
These additions look good to me, but can you double check whether this needs to also be done in `def RemoveKILevel`? It's possible that it's not needed, but that process does a lot of the same `chatArea` initialization shenanigans that these notification-based initializations are doing.

Reply:
Interesting. I don't think we need to re-do this because nothing else should be changing the KI chat area's margins.